### PR TITLE
MM-4813: Change slicing discriminator for Observation.component in zi…

### PR DESCRIPTION
**Pull request by pieter-edelman-nictiz**
…b-BloodPressure to pattern, and add guidance on using the slices

### CODE REVIEW

File: Profiles - ZIB 2017/zib-BloodPressure.xml [hunk 1]
```diff
@@ -346,117 +304,6 @@
       <path value="Observation.component.valueQuantity.code" />
       <fixedCode value="mm[Hg]" />
     </element>
-    <element id="Observation.component:AverageBloodPressureLOINC">
```
Comment (LuudSlagter) [inline Profiles - ZIB 2017/zib-BloodPressure.xml:349]: What is the reason the AverageBloodPressure components are removed completely?

File: Profiles - ZIB 2017/zib-BloodPressure.xml [hunk 2]
```diff
@@ -523,32 +355,23 @@
         <comment value="DiastolicEndpoint" />
       </mapping>
     </element>
-    <element id="Observation.component:cuffTypeLOINC">
+    <element id="Observation.component:cuffTypeSNOMED">
       <path value="Observation.component" />
-      <sliceName value="cuffTypeLOINC" />
+      <sliceName value="cuffTypeSNOMED" />
+      <comment value="There are two slices defined for the CuffType concept of zib BloodPressure (NL-CM:12.4.9), one called cuffTypeSNOMED and one called cuffTypeLOINC. These slices are defined to indicate that the `Observation.component` for this concept may be identified using either code SNOMED code 70665002 or LOINC code 8358-4. Sending systems can choose one of these codes, while receiving systems need to recognize them both. The actual _value_ communicated in `.valueCodeableConcept` is not affected by this choice.&#xD;&#xA;&#xD;&#xA;Please note that this way of modelling prevents the use of an `Observation.component` that identifies the cuff type using SNOMED code 70665002 and LOINC code 8358-4. This is a modelling error that cannot be fixed in a backwards compatible way." />
```
Comment (LuudSlagter) [inline Profiles - ZIB 2017/zib-BloodPressure.xml]: In 'code SNOMED code 70665002 or LOINC code 8358-4' the first 'code' should be removed (also in the three other comments).

Comment (LuudSlagter) [reply Profiles - ZIB 2017/zib-BloodPressure.xml]: Suggestion: add 'both' before 'SNOMED' in 'that identifies the cuff type using SNOMED code 70665002 and LOINC code 8358-4' to put extra emphasis (also in the three other comments).

**Review by LuudSlagter — CHANGES_REQUESTED** [review]

File: Profiles - ZIB 2017/zib-BloodPressure.xml (hunk 1 above)
Comment (pieter-edelman-nictiz) [reply Profiles - ZIB 2017/zib-BloodPressure.xml:349]: Probably because I'm stupid. I'll need to fix this.

Comment (pieter-edelman-nictiz) [reply Profiles - ZIB 2017/zib-BloodPressure.xml:349]: Ok, re-instated the slices and added a pattern on them.

File: Profiles - ZIB 2017/zib-BloodPressure.xml (hunk 2 above)
Comment (pieter-edelman-nictiz) [reply Profiles - ZIB 2017/zib-BloodPressure.xml]: Ok, changed it for all ~~4~~ 6 comments

**Review by LuudSlagter — APPROVED** [review]

**Pull request merged**